src/main/java/org/mule/extension/http/api/listener/HttpBasicAuthenticationFilter.java

```diff
@@ -137,6 +143,13 @@ public void authenticate(AuthenticationHandler authenticationHandler)
     }
   }
 
+  private Map<String, Object> authenticationProperties(AuthenticationHandler authenticationHandler) {
+    return authenticationHandler.getAuthentication().isPresent()
```

Comment:
This seems much simpler `authenticationHandler.getAuthentication().map(Authentication::getProperties).orElse(emptyMap())`?

Reply:
Done